source/howto/AddUsersOpenStackVMInstance.rst

```diff
@@ -0,0 +1,148 @@
+##################################################
+Create additional user(s) on OpenStack VM/Instance
+##################################################
+
+Introduction
+############
+
+This guide explains how to create users in Linux using the command line and the *useradd* command and  grant administrator privileges using *sudo*.
+
+The `sudo` command provides a mechanism for granting administrator privileges — ordinarily only available to the root user — to normal users. 
+
+This guide will show you how to create a new user with `sudo` access on Linux, without having to modify your server’s `/etc/sudoers` file.
+
+Prerequisites
+#############
+
+* Valid Login and SSH or Console Access to VM
+* Have sudo privileges
+* VPN (if outside the network)
+
+Adding Users
+############
+
+#. Logging into you server
+
+    SSH in to your server as using your FEDADMIN user:
```

Comment:
This won't be true for most users, only admins which have keys baked into the image. Users select their keypair at instance creation which get injected under `fedora` or `ubuntu` 

For SL7, it needs to be `fedora@<ip>` and for Ubuntu `ubuntu@<ip>`